webrtc.html

```diff
@@ -4088,7 +4172,33 @@ <h2>Dictionary <dfn>RTCSessionDescriptionInit</dfn>
             "RTCSessionDescriptionInit" class="dictionary-members">
               <dt><dfn data-idl><code>type</code></dfn> of type <span class=
               "idlMemberType"><a>RTCSdpType</a></span>, required</dt>
-              <dd>DOMString sdp</dd>
+              <dd>The type of this description.</dd>
+              <dt data-tests="RTCPeerConnection-setLocalDescription-rollback.html"><dfn data-idl><code>sdp</code></dfn> of type <span class=
+              "idlMemberType">DOMString</span></dt>
+              <dd>The string representation of the SDP [[!SDP]]; if
+              <code>type</code> is <code>"rollback"</code>, this member is unused.
+              </dd>
+            </dl>
+          </section>
+        </div>
+        <div>
+          <pre class="idl"
+>dictionary RTCLocalSessionDescriptionInit {
+  RTCSdpType type;
```

Comment:
The difference between RTCSessionDescriptionInit and RTCLocalSessionDescriptionInit is that "type" is not required? Can we remove the "required" bit of the original dictionary? If there is an implicit TypeError being lost when performing setRemoteDescription() or RTCSessionDescription constructor, I think I would rather add an "if type is undefined, throw TypeError" step than to introduce a near-identical dictionary for almost the same purpose.

Introducing a new dictionary to navigate around implicit IDL stuff seems less straight-forward to the reader.

Reply:
Counter-point: This shows clearly that type is required in `setRemoteDescription` but not `setLocalDescription`.

But I can try that, if people prefer it.